src/plugins/recordTypes/collectionobject/fields.js

```diff
@@ -737,6 +737,112 @@ export default (configContext) => {
         // collectionobjects_naturalhistory_extension and collectionobjects_anthro.
         ...extensions.locality.fields,
       },
+      'ns2:collectionobjects_objectcategory_extension': {
+        [config]: {
+          service: {
+            ns: 'http://collectionspace.org/services/collectionobject/domain/objectcategory_extension',
+          },
+        },
+        objectCategoryGroupList: {
+          [config]: {
+            view: {
+              type: CompoundInput,
+            },
+          },
+          objectCategoryGroup: {
+            [config]: {
+              messages: defineMessages({
+                name: {
+                  id: 'field.procedure.objectCategoryGroup.name',
```

Comment:
I think for these message ids it would be more conventional to do `collectionobjects_objectcategory_extension` instead of `procedure`.